src/test/java/com/jalasoft/jfc/controller/VideoConverterControllerTest.java

```diff
@@ -51,6 +52,7 @@
     @Before
     public void setup() {
         this.mockMvc = MockMvcBuilders.webAppContextSetup(wContext).alwaysDo(MockMvcResultHandlers.print()).build();
+        PathJfc pathJfc = new PathJfc();
     }
 
     @Test
```

Comment:
Review name of the unit test, it would be "videoConverter_WhenUploadFile_Satatus201"